server/flask_server/api.py

```diff
@@ -278,7 +278,8 @@ def get_matrix_sample(**kwargs):
     img_io.seek(0)
     return send_file(img_io, mimetype='image/png')
 
-def get_IDC_sample(id):
+def get_IDC_sample(**kwargs):
+    id = kwargs['id']
```

Comment:
To avoid a Python type error, I had to change this

Reply:
Ha, my mistake. Must accidentally left this error